pages/Transaction.tsx

```diff
@@ -53,6 +54,20 @@ export function Transaction() {
     }
   }, [walletId]);
 
+  const TransactionIcon = React.useMemo(() => {
+    if (transaction.state === "failed") {
+      return FailedTransactionIcon;
+    }
+    if (transaction.state === "pending") {
```

Comment:
pending is not the same as accepted. Accepted means your node CAN settle or cancel at any time, it currently holds the funds.

- "accepted" only applies to received payments
- "pending" only applies to sent payments

Reply:
What would be a good icon for this state 🤔